controllers/postgres_controller.go

```diff
@@ -716,11 +716,47 @@ func (r *PostgresReconciler) updatePatroniConfig(ctx context.Context, instance *
 		return err
 	}
 	if len(pods.Items) == 0 {
```

Comment:
This condition is now becoming very long and long ifs are problematic from code style perspective. Maybe it makes sense to negate it and do the entire fallback handling out of the if-condition. Other possibility would be to put this into a completely separate func.

Reply:
I agree, but I didn't want to refactor too much

Reply:
moved into separate func.